contentcuration/kolibri_public/urls.py

```diff
@@ -0,0 +1,22 @@
+from kolibri_public.views import ContentNodeTreeViewset
+from kolibri_public.views import ContentNodeViewset
+from rest_framework import routers
+
+
+public_content_v2_router = routers.SimpleRouter()
+# public_content_v2_router.register(
+#     r"channel", ChannelMetadataViewSet, basename="publicchannel"
+# )
+public_content_v2_router.register(
+    r"contentnode", ContentNodeViewset, basename="publiccontentnode"
+)
+public_content_v2_router.register(
```

Comment:
None Blocker too, but would you mind explaining why you used v2 in the naming

Reply:
Our current API endpoints are the v1 endpoints (with an exact match on Kolibri) - when these endpoints were added in Kolibri, as they were a significant update to the previous endpoints (mainly the Channel metadata one that you will be implementing), it was a significant enough change that I updated it to be the v2 endpoints rather than v1.

I could have added the ContentNode endpoints as `v1`, but that would make deprecating and eventually removing the entire `v1` public API significantly more difficult, so it seemed sensible to group all of the new public endpoints under `v2`.